### ISSUE TITLE
A11y_SQLAzureDataStudio_DeleteAzureAccount_DeleteAzureAccount_ScreenReader:The dialog is not announced when opened.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9231]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOs;#MAS1.3.1;#VoiceOver;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Mac OS:10.15.3
MAS References:MAS1.3.1
Screen reader: Voiceover.

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Connections from left menu.
4. Select Azure, navigate to + button & hit enter.
5. Launch voice over . 
6. Navigate to X button on any account in right window & hit enter.

**Actual:**
When the dialog opens the screen reader just announces the button in focus "No".

**Expected:**
The dialog should be announced when opened followed by the button.

**User Impact:**
Screen reader users will not know what "no" refers to if the name of the dialog is not announced prior.

**Attachment for referance:**
![image](https://user-images.githubusercontent.com/24580570/74928782-32f6f500-5400-11ea-8f8d-dad65c7cfbe7.png)

**Comment by alanrenmsft** [comment]
unfortunately we don't control this, the ability of openning this dialog is provided by vscode, please open an issue in vscode repo. this can be repro'ed by going to vscode and navigate to extensions view, select an extension and in the detail page, click on the repository link

<img width="1166" alt="Screen Shot 2020-03-04 at 10 59 56 PM" src="https://user-images.githubusercontent.com/13777222/75956058-f699af00-5e6b-11ea-9a18-1d3a545d39dc.png">

**Comment by BhanuPrakashGit** [comment]
Verified the bug and does not repro.
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
